### DISCUSSION TITLE
Plain text filter

### DISCUSSION THREAD

**Discussion by DipRRai** [Tyrrrz/DiscordChatExporter#1178] [Q&A]
Hello i was looking to have the export include only text ie. (without any timestamps or attachments) is there any way for me do this within the program? I wasn't sure if this question had already been answered so i'm posting this question here.

**Comment by Tyrrrz** [comment]
https://github.com/Tyrrrz/DiscordChatExporter/blob/master/.docs/Message-filters.md
